HLTrigger/Configuration/python/HLT_75e33_cff.py

```diff
@@ -314,8 +314,8 @@
 
 del fragment.offlineBeamSpot
 
-del fragment.me0RecHits
-del fragment.me0Segments
+#del fragment.me0RecHits
+#del fragment.me0Segments
```

Comment:
Shouldn't this be .. ?
```suggestion
del fragment.hltMe0RecHits
del fragment.hltMe0Segments
```
If these `del`s are not needed anymore, I would remove rather than commenting.

Reply:
I have no idea as I do not know what are the 'del' used for... How can I know what it is used for?

Reply:
@trtomei added these `del` statements, I do not know the rationale behind that. @trtomei , can you remind us?

I edited my comment above, as the suggestion is wrong, since `hltMe0Segments` is consumed by other modules.

Maybe `me0Segments` was unused even before this PR, and that's why it was being `del`eted in this file.